app/views/report_mailer/tables_report.html.erb

```diff
@@ -0,0 +1,79 @@
+<%#
+  @tables: an array of tables, expects first "row" to be an options hash
+%>
+
+<%#
+borrowed from markdown-here styling
+https://github.com/adam-p/markdown-here/blob/1df5aee09cae90e2991db4e9cfb8c1b65a3c1898/src/common/default.css
+%>
+<style type="text/css">
+table {
+  padding: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
+  font-size: 1em;
+  font: inherit;
+  border: 0;
+}
```

Comment:
@aduth I'd love some guidance on good practices around borrowing styles like this?

- Should we put them in a `.scss` file? I guess our email post-processing pipeline will already optimize away extra styles?
- Do we want to add a font to any of this?